apps/docs/scripts/search/generate-embeddings.ts

```diff
@@ -3,6 +3,9 @@ import dotenv from 'dotenv'
 import { parseArgs } from 'node:util'
 import { OpenAI } from 'openai'
 import { v4 as uuidv4 } from 'uuid'
+
+import { setupSingletonExtractor } from 'ui-patterns/CommandMenu/prepackaged/DocsSearchLocal/DocsSearchLocal.shared.llm'
```

Comment:
1) what

Reply:
that is ... a ridiculous place to put a shared util, is it not? 😂 moving...